ios/App/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>NSPrivacyAccessedAPITypes</key>
+    <array>
+      <!-- Add this dict entry to the array if the PrivacyInfo file already exists -->
+      <dict>
+        <key>NSPrivacyAccessedAPIType</key>
```

Comment:
do we need a new file? shouldnt the key be added on the current info plist here?

https://github.com/Satellite-im/UplinkWeb/blob/dev/ios/App/App/Info.plist

Reply:
For what I studied on capacitor docs, it is not allowed because it is sensitive info. 

https://capacitorjs.com/docs/apis/filesystem

And they use this Apple Doc to justify create this new file. 

https://developer.apple.com/documentation/bundleresources/describing-use-of-required-reason-api#4278393

Reply:
oh ok, thanks for explaining 😄